app/index.html

```diff
@@ -63,12 +63,10 @@ <h3 class="text-muted">EC2 Spot Pricing</h3>
               <th>Region</th>
               <th>Type</th>
               <th>Size</th>
-              <th>Price</th>
+              <th>Price <span style="position:relative;top:3px;left:1px" id="sortIcon" class="glyphicon glyphicon-triangle-top"></span></th>
```

Comment:
We can use a `caret` in place of glyphicons. They are removed in BS4 and are non-free too.